satpy/enhancements/convolution.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2017-2025 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Stretching."""
```

Comment:
Different docstring?

Reply:
Fixed in a303370dabf528f76af2d2af53fc3648fafab4ef

Reply:
Fixed in https://github.com/pytroll/satpy/pull/3155/commits/c0765feb3cc9834c62884206cd5900abbfbe5383